web/portal/src/main/webapp/groovy/portal/webui/container/UISimpleColumnContainer.gtmpl

```diff
@@ -6,6 +6,8 @@
   
   def rcontext = _ctx.getRequestContext();
   ResourceBundle res = rcontext.getApplicationResourceBundle();
+  def requirejs = rcontext.getJavascriptManager().getRequireJS();
+  requirejs.require("SHARED/bts_dropdown","bts_dropdown");
```

Comment:
sorry @boubaker your comment is deleted by mistake, in our case the UISimpleColumnContainer.gtmpl is not used in a specific portlet, it is can be added by interface in any page or site layout and we need the dropdown to insert right/left containers.